### DISCUSSION TITLE
PO Translations Format

### DISCUSSION THREAD

**Discussion by TheUltDev** [facebookincubator/fbt#243] [Ideas]
Any chance FBT could support outputting to the PO format to work with translations tools that support it. (POEdit, etc.)

**Comment by jrwats** [comment]
That is something I've been *meaning* to tackle on the side, but I really haven't been able to find the time.  If you want to take a stab at a proof of concept, I'm happy to take a look at pull requests.  See also: https://github.com/facebook/fbt/issues/60

FWIW, I think conversion to and from XLIFF (2.0) has the most promise.
